### ISSUE TITLE
Feature request: Confirmation on exit

### ISSUE THREAD

**Issue by jej** [flacon/flacon#194]
Hi all, 

If the user quits Flacon (by closing the window) during the conversion, there is no confirmation. Flacon just stops and quit.

It's confusing sometime when there are a lot of tracks to convert. If the user thinks it's finish because the visible tracks are done, he may exit Flacon while tracks outside the focus are still being converted...

I would suggest that when Flacon is converting and the user wants to exit, a confirmation window ask for:
- Stop the conversion and exit.
- Cancel and continue conversion.

Hope it helps.

**Comment by SokoloffA** [comment]
That makes sense. 
I think if conversion in progress, program should display a message like:

```
Conversion in progress. 
Are you sure you want to exit?
                    [yes] [no]
```

**Comment by jej** [comment]
LGTM.

With [no] as default.

**Comment by SokoloffA** [comment]
Sure!

**Comment by jej** [comment]
I would suggest also a double arrows spinning icon for the conversion button, that would replace the current icon as soon the conversion starts.

It would make clear that the conversion is running.

Far from C currently, so I cannot do it fluently, sorry!

**Comment by SokoloffA** [comment]
> I would suggest also a double arrows spinning icon for the conversion button, that would replace the current icon as soon the conversion starts.

I don't know what you mean. In the toolbar icon START is replaced by STOP, the rest become inactive.  What icon are you talking about?

**Comment by jej** [comment]
Sorry. I speak about the toolbar icon you mention. Maybe not a good idea, then!

**Comment by SokoloffA** [comment]
Included in release 11.0.0
